docker-compose.yml

```diff
@@ -110,7 +109,7 @@ services:
       - external
   web:
     <<: *default-rails-config
-    command: ['bin/rails', 'server']
+    command: ['bin/rails', 'server', '--binding', '0.0.0.0']
```

Comment:
Per `bin/rails server --help`:

> `-b, [--binding=IP] # Bind Rails to the specified IP - defaults to 'localhost' in development and '0.0.0.0' in other environments'.`

We need it to be `0.0.0.0` in development, as well, when running via Docker and forwarding requests through NGINX.